spec/models/product_spec.rb

```diff
@@ -0,0 +1,142 @@
+require "rails_helper"
+
+RSpec.describe Product, type: :model do
+  subject {product}
```

Comment:
subject để làm gì e ?

Reply:
subject để chỉ định  subject {product} khi dùng khối if nó tự hiểu là thằng product đấy 
vd: 
 it "belong to supplier" do
      is_expected.to belong_to :supplier
    end